statistics/handle/handle.go

```diff
@@ -1107,6 +1107,83 @@ func (h *Handle) StatsMetaCountAndModifyCount(tableID int64) (int64, int64, erro
 	return count, modifyCount, nil
 }
 
+func saveTopNToStorage(ctx context.Context, exec sqlexec.SQLExecutor, tableID int64, isIndex int, histID int64, topN *statistics.TopN) error {
```

Comment:
when you enable ```bce``` to build it, you can see the ```bce``` case.

```
go build -gcflags="-d=ssa/check_bce/debug=1" ./statistics/handle/...
```

``` 
statistics/handle/handle.go:1122:92: Found IsInBounds
statistics/handle/handle.go:1122:114: Found IsInBounds
statistics/handle/handle.go:1152:23: Found IsInBounds
statistics/handle/handle.go:1169:118: Found IsInBounds
statistics/handle/handle.go:1169:186: Found IsInBounds

```

Reply:
Can you change the code to avoid some unnecessary bounds checks?

Reply:
OK, I will fix it.